compiler/luci-interpreter/src/KernelBuilder.cpp

```diff
@@ -47,6 +48,20 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleAdd *node)
   return std::make_unique<kernels::Add>(input1, input2, output, params);
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleArgMax *node)
+{
+  assert(node->arity() == 2);
```

Comment:
Throw an exception if `node->dimension()` is not constant.
See `Reshape` and `Pad` for an example.

Reply:
Oh, Thanks! I will fix this! thanks!